taxi/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=100, unique=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=100)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE, related_name="cars_manufactured")
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="cars_driven")
```

Comment:
The second part after `_` is not needed, leave just `cars` as a related name for both `manufacturer` and `drivers`.